applications/tari_dan_wallet_daemon/src/handlers/accounts.rs

```diff
@@ -215,6 +219,7 @@ pub async fn handle_invoke(
 
 pub async fn handle_get_balances(
     context: &HandlerContext,
+    token: Option<String>,
```

Comment:
I would add the token to the context, since it will almost always be used

Reply:
Yeah, I'm working on adding jwt to the wallet_sdk so I can leave the context immutable

Reply:
The tokens are now stored in db. But I still need to send it to each command, because the token is per request.